src/components/Modals/SlotMenu.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <teleport-modal
+    content-class="content-slotmenu"
+    :isSimpleModal="true"
+    :modelValue="modelValue"
+    @modal-closed="closeCurrentModal"
+    :hasClickableTransparentBackground="true"
+    :hasCustomPosition="true"
+    :position="position"
+  >
+    <button class="slotMenu-section" @click="openEditSlotModal">
```

Comment:
The hover states are different from the course menu 
![Screen Shot 2021-05-16 at 6 26 58 AM](https://user-images.githubusercontent.com/54298311/118393947-b907e880-b60f-11eb-88da-d328adb2b502.png)
![Screen Shot 2021-05-16 at 6 27 15 AM](https://user-images.githubusercontent.com/54298311/118393955-c3c27d80-b60f-11eb-85a3-bbf6a472a3fd.png)

Try adding the class `'full-opacity-on-hover'` from global scss to these buttons and it should work.